tests/test_aperf.rs

```diff
@@ -32,6 +32,7 @@ fn test_record() {
             interval: 1,
             period: 2,
             profile: false,
+            profile_java: None,
```

Comment:
Why not add one more new test case to test Java profile ?

Reply:
We cannot test the profile because of the asprof dependency and no java workload on the machine.